client/components/ConditionCheckbox/ConditionCheckbox.js

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Checkbox from 'components/Checkbox/Checkbox';
+import styles from './ConditionCheckbox.css';
+
+export default class ConditionCheckbox extends PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      allChecked: false,
```

Comment:
what are allChecked and checkedCount for?

Reply:
I forgot to remove it. It should be removed